src/ir/control.rs

```diff
@@ -97,6 +97,8 @@ pub struct Invoke {
     pub inst: InstIdx,
     // The ports defined by this invocation
     pub ports: Vec<PortIdx>,
+    /// The events the invocation uses
+    pub events: Vec<EventBind>,
```

Comment:
Oh, this isn't quite right. It should track `EventIdx`, not `EventBind`s since the invoke defines which events are defined (in addition to how they are bound)

Reply:
Got it, will fix